### PR TITLE
FluxThis Router using ES6 generators

### PR DESCRIPTION
**Pull request by jhorwit2**
Opened this PR while I'm work is ongoing to get feedback

### CODE REVIEW

**Comment by mako-taco** [comment]
```
RangeError: Maximum call stack size exceeded.
  http://localhost:21029/test/build/tests.js:22075
  http://localhost:21029/test/build/tests.js:16174
  http://localhost:21029/test/build/tests.js:6965
[17:57:40] 'mocha-test' errored after 587 ms
[17:57:40] Error in plugin 'gulp-mocha-phantomjs'
test failed
```

**Comment by jhorwit2** [comment]
Yea, I'm not sure what the hell that is. It passes locally 100% of the time in chrome, ffx & phantomjs

**Comment by jhorwit2** [comment]
http://dev.fluxthis.io/#/docs/router

File: src/RouterStore.es6.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+				}
+			};
+		},
+		redirectTo(path) {
+			replaceHash(path);
+		},
+		navigateTo(path) {
+			window.location.hash = path;
+		},
+		getRoute(routeName, params) {
```
Comment (mako-taco) [inline src/RouterStore.es6.js]: remove

**Pull request merged**